src/porepy/numerics/interface_laws/elliptic_interface_laws.py

```diff
@@ -525,23 +501,25 @@ def assemble_edge_coupling_via_high_dim(
             ):
                 proj_flux = mg_secondary.mortar_to_secondary_int()
             # Assemble contribution between higher dimensions.
+            # import pdb
+            # pdb.set_trace()
             self.discr_primary.assemble_int_bound_pressure_trace_between_interfaces(
                 g, data_grid, proj_pressure, proj_flux, cc, matrix, rhs
             )
             # Scale the equations (this will modify from K^-1 to K scaling if relevant)
-            for block in range(cc.shape[1]):
+
+            for block in range(3):
                 # Scale the pressure blocks in the row of the primary mortar problem.
```

Comment:
primary -> secondary?

Reply:
This is the row corresponding to the mortar variable to which the contributions from the flux of block 2 is being added. I think the comment (probably written by you) refers to the two fluxes as "primary" in block 1 and "secondary" in block 2. I'll elaborate in the comment.